tests/cephfs/ceph-83573462.py

```diff
@@ -0,0 +1,195 @@
+import json
+import logging
+import random
+import string
+import time
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+
+    """
+    Pre-requisites :
+    1. Fill 60% date of the cluster
+    Test operation:
+    1. Create a volume
+    2. Mount the cephfs on both fuse and kernel clients
+    3. Create few directory from the both clients
+    4. Execute the command "ceph fs set <fs_name> max_mds n [where n is the number]"
+    5. Check if the number of mds increases and decreases properly
+    """
+    try:
+
+        tc = "CEPH-83573462"
+        log.info(f"Running CephFS tests for BZ-{tc}")
+        fs_util = FsUtils(ceph_cluster)
+        clients = ceph_cluster.get_ceph_objects("client")
+        client1 = clients[0]
+        client2 = clients[1]
+        create_cephfs = "ceph fs volume create cephfs"
+        client1.exec_command(sudo=True, cmd=create_cephfs)
+        mon_node_ips = fs_util.get_mon_node_ips()
+        kernel_dir_generate = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(5))
+        )
+        kernel_mounting_dir = f"/mnt/cephfs_kernel{kernel_dir_generate}/"
+        fs_util.auth_list([client1])
+        fs_util.kernel_mount([client1], kernel_mounting_dir, ",".join(mon_node_ips))
+        client1.exec_command(
+            sudo=True,
+            cmd=f"dd if=/dev/zero of={kernel_mounting_dir}" + ".txt bs=5M count=1000",
+            long_running=True,
+        )
+        for i in range(10):
+            dir_name_generate = "".join(
+                random.choice(string.ascii_lowercase + string.digits)
+                for _ in list(range(5))
+            )
+            client1.exec_command(
+                sudo=True,
+                cmd="mkdir %s%s" % (kernel_mounting_dir, "dir_" + dir_name_generate),
+            )
```

Comment:
can you use f-string
client1.exec_command(
                sudo=True,
                cmd=f"mkdir {kernel_mounting_dir}dir_{dir_name_generate}" 
            )

Reply:
Okay. thanks